atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -344,6 +373,26 @@ def get_kojifile_source_urls(self):
 
         return kojifile_sources
 
+    def get_remote_file_urls(self):
+        """Fetch remote source file urls from all builds
+
+        :return: list, dicts with URL pointing to remote source files
+        """
+        sources = []
+
+        source = self._get_remote_file_urls_helper(self.koji_build)
+        sources.extend(source)
+
+        koji_build = self.koji_build
+
+        while 'parent_build_id' in koji_build['extra']['image']:
+            koji_build = self.session.getBuild(koji_build['extra']['image']['parent_build_id'],
```

Comment:
Renaming `koji_build` to `parent_koji_build` to not to overwrite different object?

Reply:
we aren't overwriting different object but we still need the first build for while loop which is from : 
koji_build = self.koji_build

Reply:
got it now